### PR TITLE
Acknowledge MARVEL and MaX; convert README.rst to README.md

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+[![image](https://travis-ci.org/aiidateam/aiida_core.svg?branch=develop)](https://travis-ci.org/aiidateam/aiida_core)
```
Comment (ltalirz) [inline README.md]: could you please update the placeholders to something meaningful? It's already done in the link I provided.

Unfortunately, it happens quite frequently that one of the badges isn't loaded.

**Comment by ltalirz** [comment]
@giovannipizzi or @sphuber : please check and squash+merge

**Comment by sphuber** [comment]
Out of curiosity, what is the reason for moving from `.rst` to `.md`?

**Comment by ltalirz** [comment]
We are using markdown across the board (plugins, aiidalab apps, demos, ...).
Since he needed to apply the same acknowledgements in multiple places, it just made sense to switch.

**Comment by muhrin** [comment]
@yakutovicha @ltalirz

Sorry to dive in late but I've got a slight concern here.  Markdown has only recently began to be rendered by PyPi and even then it requires a few steps.  Personally I would favour keeping reST but if you insist on MD it would be good follow the instructions here:

https://stackoverflow.com/questions/26737222/pypi-description-markdown-doesnt-work

**Comment by ltalirz** [comment]
Sure, it is already set up properly, the only think to keep in mind is to use

* twine >= 1.11.0
* wheel >= 0.31.0

happy to add that to the wiki

**Pull request merged**